docs/index.md

```diff
@@ -4103,6 +4105,109 @@ return api.getLedger()
 ```
 
 
+## parseAccountFlags
+
+`parseAccountFlags(Flags: number): object`
+
+Parse an `AccountRoot` object's [`Flags`](https://developers.ripple.com/accountroot.html#accountroot-flags).
+
+### Parameters
+
+This method takes one parameter, the `Flags` number to parse.
```

Comment:
Maybe worth a note that this only works for AccountRoot `Flags` since the flags have different mappings on other types of objects or in transactions like AccountSet.